lnbits/core/static/js/wallet.js

```diff
@@ -795,6 +795,11 @@ new Vue({
         field: 'pending'
       })
       LNbits.utils.exportCSV(columns, this.payments)
+    },
+    pasteToTextArea: function () {
+      navigator.clipboard.readText().then(text => {
```

Comment:
Just fyi: `navigator.clipboard` is not available on sites that are either

- not localhost
- or not HTTPs

Some people might access their lnbits behind a VPN which is a secure connection but HTTP or a LAN; should be considered to implement a fallback for that scenario or hide the button


https://stackoverflow.com/a/65996386